caikit/runtime/http_server/http_server.py

```diff
@@ -1002,7 +1188,7 @@ def _get_response_openapi(
             for def_name, schema in json_schema.pop("$defs", {}).items():
                 self._openapi_defs[def_name] = schema
 
-            response_schema = {"application/json": json_schema}
+            response_schema = {"application/json": {"schema": json_schema}}
```

Comment:
Interesting. What was this line about?

Reply:
Oh this is a change that should probably be its own PR but I snuck it in here..... all of our openapi generation is broken rn since it has the incorrect schema format 😅

Reply:
Deal. You _might_ want to split this into its own PR so it can be merged quickly